### ISSUE TITLE
Wing Pointe Dr

### ISSUE THREAD

**Issue by gregbunce** [agrc/api.mapserv.utah.gov#52]
"Wing Pointe" Dr will not geocode in the api geocoders as "Wing Point" Dr
sample (real world) address that codes: 3501 S Wing Pointe Dr Salt Lake City
sample (not real world) address that does not code: 3501 S Wing Point Dr Salt Lake City

**Comment by steveoh** [comment]
this is an invalid address and should not geocode. Thank you for the heads up!

**Comment by steveoh** [comment]
@BGranberg would like to create an exception for this in the alias table without the `e` as a common misspelling.

**Comment by steveoh** [comment]
@gregbunce can you look into a way of adding exceptions to the roadgrinder project and add this one. I will assign this to you for now. Thanks!

**Comment by gregbunce** [comment]
@steveoh certainly.  i can see possibly needing this option for other roads as well.  i'll check it out.

**Comment by gregbunce** [comment]
Done.  I added "Wing Point" as an Alias name for the street "Wing Pointe".  Next time we run the RoadGrinder (mid to late Oct) it will pick up this update and add that alias to the RoadGrinder's altnames table for those 6 segments.  Addresses on those streets will then geocode with "Wing Point" or "Wing Pointe".  I made this alternate name change in the reference data, as it seems best suited there and will then persist.
